libraries/AP_Param/AP_Param.cpp

```diff
@@ -208,7 +210,7 @@ bool AP_Param::check_frame_type(uint16_t flags)
 }
 
 // validate a group info table
-bool AP_Param::check_group_info(const struct AP_Param::GroupInfo *  group_info,
+void AP_Param::check_group_info(const struct AP_Param::GroupInfo *  group_info,
```

Comment:
these parameter changes should be a separate PR

Reply:
They are already :-) https://github.com/ArduPilot/ardupilot/pull/22597